DisCatSharp/Clients/DiscordClient.cs

```diff
@@ -488,40 +488,40 @@ public async Task<DiscordApplication> GetCurrentApplicationInfoAsync()
 	/// Tries to get a user.
 	/// </summary>
 	/// <param name="userId">Id of the user.</param>
+	/// <param name="user">The user, if found.</param>
 	/// <param name="fetch">Whether to ignore the cache. Defaults to true.</param>
-	/// <returns>The requested user or null if not found.</returns>
-	/// <exception cref="BadRequestException">Thrown when an invalid parameter was provided.</exception>
-	/// <exception cref="ServerErrorException">Thrown when Discord is unable to process the request.</exception>
-#pragma warning disable CS8632 // The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
-	public async Task<DiscordUser?> TryGetUserAsync(ulong userId, bool fetch = true)
-#pragma warning restore CS8632 // The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
+	/// <returns>True if found, otherwise false.</returns>
+	public bool TryGetUserAsync(ulong userId, out DiscordUser user, bool fetch = true)
```

Comment:
These should still be `async Task<bool>`, they'll do requests after all.

Reply:
C# for some reason won't allow async tasks when you have out var parameters. That's the error I got at least when writing it. 
The methods are still async, they will just be awaited automatically because I can't imagine a case where you want to request something and not wait for it.

Reply:
Oh, i didn't even know that. Then remove the Async Suffix please

Reply:
That might also be the reason it originally returned null instead of the bool, out var pattern.

Reply:
Yeah, this is the error btw: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs1988
also done